REQUIRE

```diff
@@ -1,5 +1,5 @@
 julia 0.5
-DataArrays 0.3.4
+DataArrays 0.5
```

Comment:
The problem is that DataArrays only supports Julia 0.6, but DataFrames still supports 0.5. That's why the 0.5 CI builds are failing.